var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -18,20 +18,21 @@ class Molgw(MakefilePackage):
     """
 
     homepage = "https://github.com/molgw/molgw"
-    url = "https://github.com/molgw/molgw/archive/v3.3.tar.gz"
+    url = "https://github.com/molgw/molgw/archive/v3.4.tar.gz"
     git = "https://github.com/molgw/molgw.git"
 
     maintainers("bruneval")
 
     license("GPL-3.0-only")
 
+    version("3.4", sha256="b7ce298c660861713c25873d46e09713cad80b3927110a6bf747c0114b387e6b")
     version("3.3", sha256="ff1c8eb736049e52608d4554a2d435ee9d15e47c4a9934d41712962748929e81")
     version("3.2", sha256="a3f9a99db52d95ce03bc3636b5999e6d92b503ec2f4afca33d030480c3e10242")
 
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    variant("openmp", default=False, description="Build with OpenMP support")
+    variant("openmp", default=True, description="Build with OpenMP support")
```

Comment:
```suggestion
    variant("openmp", default=False, description="Build with OpenMP support")
```

It is best practice to have variants be `default=False` not `True` as this increases installation sizes by default requires users use `~openmp` when not needed which causes a hard conflict for packages that require openmp. Can you explain why this variant required by default?

Reply:
No specific reason. I'll revert this change, as you have strong arguments against it.